test/frame.test.js

```diff
@@ -0,0 +1,33 @@
+const Frame = require('../lib/frame');
+
+describe('Frame', () => {
+  it('Check create new instant', () => {
+    const frame = new Frame();
+    expect(frame.getFrameResult()).toEqual([]);
+  });
+
+  it('Add score to frame result', () => {
+    const frame = new Frame();
```

Comment:
I wonder if you could have defined the 'const frame 'once under at the start, then you wouldnt need to repeat this line in each test?